2D Simulation/game.js

```diff
@@ -76,6 +77,54 @@ function getGameType() {
     }
 }
 
+/////////////////////
+// IncorrectButton //
+/////////////////////
+
+function get_incorrect_button() {
+    return $("button#buttonIncorrect");
+}
+
+function setup_incorrect_button() {
+    let incorrect_button = get_incorrect_button();
+
+    if (getGameType() !== "ai") {
+        incorrect_button.css({
+            visibility: "hidden"
+        });
```

Comment:
Does it break anything to replace this with `.hide()`?

Reply:
Nope!